extension.js

```diff
@@ -44,7 +19,7 @@ function gremlinsFromConfig(context) {
       light: lightIcon,
       dark: darkIcon,
       overviewRulerColor: config.overviewRulerColor || 'darkred',
-      overviewRulerLane: vscode.OverviewRulerLane.Right,
+      overviewRulerLane: vscode.OverviewRulerLane.Right
```

Comment:
We use Prettier to format our code, and like the trailing commas:
https://github.com/nhoizey/vscode-gremlins/blob/master/package.json#L60-L64
https://prettier.io/docs/en/options.html#trailing-commas

Didn't your IDE use Prettier when saving your edits?